src/main/java/io/github/enoughsdv/refill/utils/MessageUtil.java

```diff
@@ -1,21 +1,25 @@
 package io.github.enoughsdv.refill.utils;
 
-import java.util.List;
 import java.util.ArrayList;
+import java.util.List;
+
+import org.bukkit.ChatColor;
 
 public class MessageUtil {
     
     private MessageUtil() {
         throw new java.lang.UnsupportedOperationException("This is a utility class and cannot be instantiated");
     }
-    
+
     public static String translate(String message) {
-        return org.bukkit.ChatColor.translateAlternateColorCodes('&', message);
+        return ChatColor.translateAlternateColorCodes('&', message);
     }
 
-    public static List<String> translate(List<String> messages) {
-        List<String> buffered = new ArrayList<>();
-        messages.forEach(message -> buffered.add(translate(message)));
+    public static List<String> translateStrings(List<String> message) {
+        List<String> buffered = new ArrayList<String>();

```

Comment:
The plugin is compatible with java 8+, so it is no longer necessary to add it so I recommend the use of diamond difference.